### ISSUE TITLE
Date picker dialog example - Feedback from APG list

### ISSUE THREAD

**Issue by lolaodelola** [w3c/aria-practices#3151]
"Hello

The date picker dialog example has interactive cells but they do not contain interactive UI components like a button for example. How is a screen reader user supposed to know the cells are interactive? Especially if you compare it to other grid examples such as the 'data grid' example where some grid cells are interactive and some aren't, but you can distinguish between them because it isn't the cell itself that's interactive, but rather it contains interactive content such as a link.

Also the example doesn't include how to handle dates you may wish to disable - dates in the past are often disabled in a date picker. This is another reason why it would be better to have buttons inside the grid cells which could be marked up as disabled surely?

Thanks

Sarah"
- July 31st 2024

**Comment by dd8** [comment]
The date picker uses `aria-selected=true` on the currently selected date cell, but does not have any `aria-selected` attributes on the  unselected date cells, which means they're unselectable according to the spec:

false | The selectable element is not selected.
true | The selectable element is selected.
undefined (default) | The element is not selectable.

https://www.w3.org/TR/wai-aria-1.3/#aria-selected

Should the unselected cells use `aria-selected=false` ?
